app/models/lookup_keys/lookup_key.rb

```diff
@@ -103,7 +103,8 @@ def supports_uniq?
   end
 
   def to_param
-    Parameterizable.parameterize("#{id}-#{key}")
+    search_key = key.tr(' ','_') unless key.nil?
```

Comment:
I think we don't need a down migration, any parameter with spaces was invalid, no need to restore them. (and I think this reply was meant for the migration comment?)